cmd/geth/chaincmd.go

```diff
@@ -446,10 +447,10 @@ func copyDb(ctx *cli.Context) error {
 	if len(ctx.Args()) < 2 {
 		utils.Fatalf("Source ancient chain directory path argument missing")
 	}
-	// Initialize a new chain for the running node to sync into
-	stack, _ := makeFullNode(ctx)
-	defer stack.Close()
 
+	stack, _ := makeConfigNode(ctx)
+	defer stack.Close()
+	// Initialize a new chain for the running node to sync into
```

Comment:
Code style nit, pls leave an empty line before a comment. Loosens the code up a bit :)

Reply:
Here again imho the original location was better. Mind you, you don't need an empty line there because `}` on itself is already loose enough :)